backends/gaudi/Makefile

```diff
@@ -2,7 +2,7 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 mkfile_dir := $(dir $(mkfile_path))
 root_dir := ${mkfile_dir}/../..
 
-HABANA_VERSION := 1.21.0
+HABANA_VERSION := 1.20.0
```

Comment:
Can you explain why you're downgrading it?

Reply:
Sorry, I use the 1.20 firmware. Remove the modification.